### PR TITLE
Add Arnis

### PR DESCRIPTION
**Pull request by louis-e**
Arnis is a Python project using OSM to generate cities from real life in Minecraft - experimental, but might be interesting enough to be added to the list :)

### CODE REVIEW

**Review by mnalis — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -185,6 +185,7 @@ This section is a great place to start if you want to get into improving OpenStr
 
 * [Back Of Your Hand](https://backofyourhand.com/) - A web map game that tests your knowledge by having you find a street in a given area. ([Source Code](https://github.com/adam-lynch/back-of-your-hand))
 * [PraxisMapper](https://github.com/drakewill-CRL/PraxisMapper) - GPS Game server, for making mobile games around exploring the real world. ([Source Code](https://github.com/drakewill-CRL/PraxisMapper))
+* [Arnis](https://github.com/louis-e/arnis) - Generate cities from real life in Minecraft using Python. ([Source Code](https://github.com/louis-e/arnis))
```
Comment (Unkn0wnCat) [inline README.md:188]: For projects with only a GitHub link we omit the Source Code link, as duplicated links are bad practise.

```suggestion
* [Arnis](https://github.com/louis-e/arnis) - Generate cities from real life in Minecraft using Python.
```

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Hey @louis-e, thanks for your contribution. It already looks great, although I have one minor change I need you to apply before I can merge it. (See attached comment)

It seems the automatic linter has gone a bit haywire, so don't mind the failed action on the PR - This is not related to your PR and I'm looking into fixing it!

**Comment by louis-e** [comment]
Alright, thanks for the notice! I also removed the unnecessary part in the previous line (PraxisMapper).

**Comment by Unkn0wnCat** [comment]
Great! Thanks for the addition (and the fix of the line above 😄).

Your project is really awesome and will fit right in on the list 🎉

**Pull request merged**